tasks/taxon_id/task_kraken2.wdl

```diff
@@ -140,4 +145,84 @@ task kraken2_standalone {
       disks: "local-disk 100 SSD"
       preemptible: 0
   }
+}
+
+task kraken2_parse_classified {
+  input {
+    File kraken2_classified_report
+    File kraken2_report
+    String samplename
+    String? target_org
+    Int cpu = 4
+    Int disk_size = 100
+  }
+  command <<<
+
+    gunzip -c ~{kraken2_classified_report} > ~{samplename}.classifiedreads.txt
+
+    python3 <<CODE 
+    import pandas as pd 
+
+    # load files into dataframe for parsing
+    classifiedreads_table = pd.read_csv("~{samplename}.classifiedreads.txt", names=["Classified_Unclassified","Read_ID","Taxon_ID","Read_Len","Other"], header=None, delimiter="\t")
```

Comment:
if you are the one making these headers, i would recommend lowercase

Reply:
done!